libs/gperftools/Makefile

```diff
@@ -0,0 +1,80 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gperftools
+PKG_VERSION:=
```

Comment:
?

Reply:
I modeled this after another git Makefile where `PKG_VERSION:=` was undefined.  I assume it was needed.

Reply:
I switched to a tag now that the developer included one